packages/libp2p/test/connection-manager/index.node.ts

```diff
@@ -235,7 +235,9 @@ describe('libp2p.connections', () => {
           },
           connectionManager: {
             minConnections,
-            maxConnections: 1
+            maxConnections: 1,
+            inboundConnectionThreshold: 1,
```

Comment:
Why do we need to specify the extra config now?  This seems to happen in a few places here.

Reply:
Because the intention is that we should not be able to have more inbound connections than `maxConnections` as that would result in them getting pruned anyways, discussing with @wemeetagain he raised a use case where this may be applicable though, where a node may want to reach maxConnections as fast as possible and so it may accept a greater amount of inbound connections in a short space of time.